ci/build.yml

```diff
@@ -109,7 +109,8 @@ jobs:
         bazel build //release:release
         ./bazel-bin/release/release --release-dir "$(mktemp -d)"
       env:
-        DAML_SCALA_VERSION: ${{parameters.scala_version}}
+        DAML_SCALA_VERSION: "2.13.3"
```

Comment:
I pushed another fix here since the step which now triggers after fixing the check didn’t build against 2.13.

Reply:
Can you add a comment explaining this in the file?

Reply:
explaining what? that referencing a non-existent variable doesn’t work? that seems a bit pointless.

Reply:
Nah, just explaining that this is a duplicate of another constant elsewhere and that they need to be kept in sync. (And in the other place.)